clients/admin-ui/src/types/dictionary-api/index.ts

```diff
@@ -2,12 +2,10 @@
 /* tslint:disable */
 /* eslint-disable */
 
-export type { ContactDetails } from "./models/ContactDetails";
```

Comment:
not directly related to this change, but these came up as changes when running the dictionary generate command. they're no longer present/active on the compass side, so figured it was good to cleanup as some maintenance

Reply:
👍